test/sql/copy/csv/test_csv_time_format_errors.test

```diff
@@ -0,0 +1,25 @@
+# name: test/sql/copy/csv/test_csv_time_format_errors.test
+# description: Test reading csv files containing time format errors
+# group: [csv]
+
+statement error
```

Comment:
Could you add 
```
statement ok
PRAGMA enable_verification
```

to the top of all sql tests and make sure the new option is properly serialized?